rmw_implementation/package.xml

```diff
@@ -15,14 +15,13 @@
   <depend>libpoco-dev</depend>
   <depend>poco_vendor</depend>
   <depend>rmw_implementation_cmake</depend>
-  <depend>rmw_connext_cpp</depend>
-  <depend>rmw_connext_dynamic_cpp</depend>
   <depend>rmw_fastrtps_cpp</depend>
-  <depend>rmw_opensplice_cpp</depend>
```

Comment:
Implicitly the decision here is to release with an explicit dependency on fastrtps only. I guess if we build binaries with support for more we'd add the keys back here?

Perhaps a comment on the fastrtps dependency, indicating that it is there for packaging purposes and is redundant with the group dependency when building from source, would help people interpreting the `package.xml` in the future.

Reply:
:+1: the absence of comment here is what led to my question of "if a user want to modify the defaut rmw_imp and build only for that one, do they need to modify this manifest?"

Reply:
> Implicitly the decision here is to release with an explicit dependency on fastrtps only. I guess if we build binaries with support for more we'd add the keys back here?

Yes, this reflects what we did release in the past betas.

> Perhaps a comment on the fastrtps dependency, indicating that it is there for packaging purposes and is redundant with the group dependency when building from source, would help people interpreting the `package.xml` in the future.

Can you please either propose a wording or commit the comment directly. Thanks.

> if a user want to modify the defaut rmw_imp and build only for that one, do they need to modify this manifest

Nothing has changed here. The user places whatever rmw impl he wants to build into the workspace and if it is more than one and he doesn't use the default (which is FastRTPS if present otherwise the first in alphabetical order) it can be overridden by specifying the environment variable `RMW_IMPLEMENTATION`.